plugins/octant/cmd/antrea-octant-plugin/traceflow.go

```diff
@@ -194,17 +241,29 @@ func (p *antreaOctantPlugin) actionHandler(request *service.ActionRequest) error
 		if err != nil {
 			log.Printf("Invalid user input, CRD creation or Traceflow request may fail: "+
 				"failed to get srcPort as int: %s", err)
+			alert := action.CreateAlert(action.AlertTypeError, fmt.Sprintf("Failed to get source port as "+
```

Comment:
I think source port and destination port do not need to be set when users choose ICMP. 
We should not just return for these case.  Could you please double check this part?

Reply:
Ports are not necessary for all protocols. I have updated this part. If port numbers are invalid (empty, negative, too big, etc.), we will use random port numbers (TCP) or 0 (UDP) in the backend.

Reply:
For your current implementation, UI can allow Traceflow CRD creation with empty source/destination port for TCP protocol. I am not sure if Traceflow agent can handle this and think it may cause problem. @gran-vmv Ran, could you review and confirm this?